setup.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import os
+from os.path import join
+from distutils.sysconfig import get_python_lib
+
+from setuptools import find_packages
+from skbuild import setup
+from skbuild.constants import CMAKE_INSTALL_DIR
+import re
+
+# read the version number from the library
+pattern = r"[0-9]\.[0-9]\.[0-9]"
+version = None
+with open('./bindings/rascal/__init__.py', 'r') as fp:
+    for line in fp.readlines():
+        if '__version__' in line:
+            version = re.findall(pattern, line)[0]
+if version is None:
+    raise ValueError("Version number not found.")
+
+with open('./requirements_pip.txt', 'r') as fp:
+    requirements = list(filter(lambda x: '#' not in x, (line.strip() for line in fp)))
+
+setup(
+    name="rascal",
+    version=version,
+    description="""A versatile and scalable computation of representations of
+atomic structures for machine learning.""",
+    author='Félix Musil',
+    license="LGPL-3",
```

Comment:
I have never seen the license specified as an argument to setup, instead it is usually in `classifiers`, like `"License :: OSI Approved :: Python Software Foundation License"`. I could also not find documentation for this specific keyword.

Reply:
this is a proper keyword as you can see from [here](https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=setup#configuring-setup-using-setup-cfg-files). classifiers is another metadata keyword that serves another purpose.

Reply:
Ok, good! Should it be using SPDX license identifier, i.e. `LGPL-3.0-or-later` here?

Reply:
https://spdx.org/licenses/

Reply:
we use `LGPL-3.0` so I would use `LGPL-3.0-only` rather than what you suggest. Or am I missing something ?

Reply:
`LGPL-3.0-only` works, I did not check if we had the `or any later version` text in the license

Reply:
Should actually be the `-or-later` version

```
# Rascal is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public License as
# published by the Free Software Foundation, either version 3, or (at
# your option) any later version.
```